Dockerfile

```diff
@@ -18,6 +18,7 @@ ADD ./calico_isolator.py /isolator/
 ADD ./configure.ac /isolator/
 ADD ./Makefile.am /isolator/
 ADD ./requirements.txt /isolator/
+ADD ./calico-node.tar /isolator/
```

Comment:
This breaks the build.  We'll have to update CI to use the Makefile instead of doing docker build (not totally ideal).  Verifying manually.